.github/workflows/build-preflight-in-a-box.yml

```
@@ -0,0 +1,107 @@
+name: New Tag
+on: push
+
+jobs:
+  build:
+    # This job only runs for issue comments
+    name: Update package
+    runs-on: ubuntu-latest
+    #     permissions:
+    #       contents: read
+    #       packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set Node.js version 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```

To keep this consistent with our other actions

```suggestion
      - name: Use Node.js
        uses: actions/setup-node@v2
        with:
          node-version: '16'
```